src/App/index.jsx

```diff
@@ -1,6 +1,4 @@
-import Benchmark from '../views/Benchmark';
+import Routes from '../routes';
 import './metricsGraphics.css';
 
-const App = () => <Benchmark />;
-
-export default App;
+export default () => <Routes />;
```

Comment:
Given the simplicity of the Routes component, I would just inline it directly into this App component:

```jsx
export default class App extends Component {
  render() {
    return (
      <Switch>
        <Route path="/" component={Benchmark} />
      </Switch>
    );
  }
}
```